lumicks/pylake/kymotracker/tests/test_msd.py

```diff
@@ -0,0 +1,164 @@
+from lumicks.pylake.kymotracker.detail.msd_estimation import *
+import contextlib
+import pytest
+
+
+@contextlib.contextmanager
+def temp_seed(seed):
+    np.random.seed(seed)
+    try:
+        yield
+    finally:
+        np.random.seed(None)
+
+
+def simulate_diffusion_1d(diffusion, steps, dt, observation_noise):
+    """Simulate from a Wiener process
+
+    Parameters
+    ----------
+    diffusion : float
+        Diffusion constant.
+    steps : int
+        Number of steps to simulate.
+    dt : float
+        Time step.
+    observation_noise : float
+        Standard deviation of the observation noise.
+    """
+
+    def simulate_wiener(sigma, num_steps, time_step):
+        return np.cumsum(np.random.normal(0, sigma * np.sqrt(time_step), size=(num_steps,)))
+
```

Comment:
does this have to be a separate function, since it's only used once in the next line?

Reply:
I thought it helped to show the relation between the standard Wiener process and Brownian motion (quite often the two are used interchangeably), but I could get rid of it if you want.